src/sources/vector/mod.rs

```diff
@@ -50,6 +53,17 @@ impl proto::Service for Service {
             .map(Event::from)
             .collect();
 
+        let now = Utc::now();
+        for event in &mut events {
+            if let Event::Log(ref mut log) = event {
+                self.log_namespace.insert_standard_vector_source_metadata(
+                    log,
+                    VectorConfig::NAME,
```

Comment:
I can't think of what we _might_ add that would be more "source metadata" vs "vector metadata" on this component, but it's worth remembering the name will be the same for both.

Reply:
That is an excellent point.

Reply:
@fuchsnj @jszwedko can y'all foresee any issues with this?

Reply:
We discussed as a team and while it is interesting to note, we felt it would be ok even if we did add to the "source" metadata later on, since it is all for vector.